pkg/cluster/cache_whitebox_test.go

```diff
@@ -425,7 +369,7 @@ func TestMultipleActionsInParallel(t *testing.T) {
 			go func() {
 				defer waitForFinished.Done()
 				latch.Wait()
-				clusters := clusterCache.getCachedToolchainClustersByType(clusterToTest.Type)
+				clusters := clusterCache.getCachedToolchainClusters()
```

Comment:
here we should expect: that all the returned toolchainclusters are the in the expected list of toolchaincluster or the list can be empty. WDYT?

Reply:
i think this will be dependent on what we are passing at "clusterCache.addCachedToolchainCluster(clusterForTest)"  , here i removed  the other types bcoz, it becomes difficult to test through member and host, as the function here would bring in all the clusters, we can do if else but that wouldnt be so good, i think to test whether the function getCachedToolchainClusters() works good, passing one is good.. your thoughts..?

Reply:
ok, makes sense if we use just one toolchaincluster resource. Thanks for explaining that!